t/bm_move_spec.vim

```diff
@@ -0,0 +1,176 @@
+
+" custom matchers
```

Comment:
Great work with the tests/spec! Thanks for covering you changes properly by tests 👍